src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -459,9 +459,15 @@ sampleGetBigBook(argv.shelf);
 // [START hopper]
 // [START hopper_core]
 
-
 const library = require('@google-cloud/library').v1;
 
+/**
+ * Testing resource name overlap
+ *
+ * @param shelf {string} Test word wrapping for long lines. This is a long
+ * comment. The name of the shelf to retrieve the big book from.
```

Comment:
IMO, it would be nice to indent lines after the first in param descriptions (open to guidance from DevRel). I guess for Java we post-run the formatter, but what about other languages? And to properly account for line length with the indentation, we should probably indent the lines in the generator before wrapping them, even if we later run a formatter.

If it's not simple to do this now, let's file an issue and get back to it separately. I think this is a cosmetic issue we'll need to fix, but no need to block this PR.

Reply:
It's not simple to do, and also our clients do not have indentations when param descriptions take up more than one line: https://github.com/googleapis/gapic-generator/blob/96c3c5a4c8397d4bd29a6abce861547a271383e1/src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline#L1738.

So I thought it'd be OK.